fh-sync-js.d.ts

```diff
@@ -2,237 +2,190 @@
 // Project: https://github.com/feedhenry/fh-sync-js
```

Comment:
You will make hart time to merge @evanshortiss PR after: https://github.com/feedhenry/fh-sync/pull/41/files
I think it's best to not add this changes to this PR (it should be targeting single problem) and contribute type changes to PR above.

Reply:
Strictly speaking, these changes *are* necessary though, as the factory function is not exposed without these changes.

Also, Evan's changes are for the cloud code, so I'm not sure how there will be a conflict?

Reply:
My bad. Sorry, got it mixed by names. Feel free to ignore this.

Reply:
:) Evan and I were chatting about the typings last night actually, and we were getting mixed up ourselves. I guess there's no chance of changing the repo names to be something like `fh-sync-client` and `fh-sync-server`? :P

Reply:
fh-sync-client is not a way to go as we have multiple platforms. For example: https://github.com/feedhenry/fh-sync-android

Reply:
`fh-sync-client-js`? The fact that fh-sync is also JS kind of indicates to me that we need a better qualifier for the JS client code than `-js`.

Reply:
That sounds ok. I will run that on mailing list.

Reply:
Granted, that would create an inconsistency in naming unless you also renamed the client repos. `fh-sync-client-android`, etc. Renaming `fh-sync` to `fh-sync-server` would probably be sufficient.